test_tar_changes.py

```diff
@@ -75,3 +75,9 @@ def test_upload_source(delete_sources, mocked_run, mocked_sources):
     # flag was detected by argparse
     tarchanges.upload_source(osdist, 'test.tar', False)
     mocked_run.assert_called_once() # run() was not called a 2nd time
+
+def test_format_changelog():
+    # Check of cases when bzlist has values and is empty
+    actual = tarchanges.format_changelog([("9e20ef1b14ac70dea53123", "cephbz" ,["123","567"]),("9e20ef1b14ac70dea53456", "bzceph" ,[])])
+    expected = ["cephbz (rhbz#123 rhbz#567)","bzceph"]
+    assert actual == expected
```

Comment:
Lets make these 2 explicit assertions, in case this fails it'll be immediately obvious which assertion failed

Reply:
Made the suggested changes